README.md

```diff
@@ -239,30 +317,84 @@ can determine the _order_ in which they need to be worked on.
 
 ### C) _Estimate_ the Time Required to Perform the Work
```

Comment:
@iteles yes, that needs to be _clarified_. it should be the people _doing_ the work who estimate it. let me _move_ this down now. 👍